pkg/resource/training_job/testdata/v1alpha1/readone/observed/stopping.yaml

```diff
@@ -0,0 +1,107 @@
+      regex: .*\[[0-9]+\].*#011train-logloss:([-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?).*
+    - name: train:mlogloss
+      regex: .*\[[0-9]+\].*#011train-mlogloss:([-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?).*
+    - name: validation:rmse
+      regex: .*\[[0-9]+\].*#011validation-rmse:([-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?).*
+    - name: validation:ndcg
+      regex: .*\[[0-9]+\].*#011validation-ndcg:([-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?).*
+    - name: train:error
+      regex: .*\[[0-9]+\].*#011train-error:([-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?).*
+    - name: validation:mlogloss
+      regex: .*\[[0-9]+\].*#011validation-mlogloss:([-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?).*
+    - name: train:ndcg
+      regex: .*\[[0-9]+\].*#011train-ndcg:([-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?).*
+    - name: train:map
+      regex: .*\[[0-9]+\].*#011train-map:([-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?).*
+    - name: validation:map
+      regex: .*\[[0-9]+\].*#011validation-map:([-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?).*
+    trainingImage: 433757028032.dkr.ecr.us-west-2.amazonaws.com/xgboost:1
+    trainingInputMode: File
+  enableInterContainerTrafficEncryption: false
+  enableManagedSpotTraining: false
+  enableNetworkIsolation: false
+  hyperParameters:
+    eta: "0.2"
+    gamma: "4"
+    max_depth: "5"
+    min_child_weight: "6"
+    num_class: "10"
+    num_round: "10"
+    objective: multi:softmax
+    silent: "0"
+  inputDataConfig:
+  - channelName: train
+    compressionType: None
+    contentType: text/csv
+    dataSource:
+      s3DataSource:
+        s3DataDistributionType: FullyReplicated
+        s3DataType: S3Prefix
+        s3URI: s3://source-data-bucket-592697580195-us-west-2/sagemaker/training/train
+    recordWrapperType: None
+  - channelName: validation
+    compressionType: None
+    contentType: text/csv
+    dataSource:
+      s3DataSource:
+        s3DataDistributionType: FullyReplicated
+        s3DataType: S3Prefix
+        s3URI: s3://source-data-bucket-592697580195-us-west-2/sagemaker/training/validation
+    recordWrapperType: None
+  outputDataConfig:
+    kmsKeyID: ""
+    s3OutputPath: s3://source-data-bucket-592697580195-us-west-2/sagemaker/training/output
+  resourceConfig:
+    instanceCount: 1
+    instanceType: ml.m4.xlarge
+    volumeSizeInGB: 5
+  roleARN: arn:aws:iam::123456789012:role/service-role/AmazonSageMaker
+  stoppingCondition:
+    maxRuntimeInSeconds: 86400
+  tags:
+  - key: algorithm
+    value: xgboost
+  - key: environment
+    value: testing
+  - key: customer
+    value: test-user
+  trainingJobName: xgboost-training-job
+status:
+  ackResourceMetadata:
+    arn: arn:aws:sagemaker:us-west-2:123456789012:training-job/xgboost-training-job
+    ownerAccountID: ""
+  conditions:
+  - lastTransitionTime: "0001-01-01T00:00:00Z"
+    message: TrainingJob is in Stopping status.
+    status: "False"
+    type: ACK.ResourceSynced
+  secondaryStatus: Starting
+  trainingJobStatus: Stopping
```

Comment:
How did this unit test pass considering we have "Completed" in desired state and "Stopping" in observed state ?